dbms/src/IO/CodecDeflateQpl.h

```diff
@@ -0,0 +1,82 @@
+// Copyright 2022 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef SRC_LIB_COMPRESS_QPL_H_
```

Comment:
I think this header needs to be enclosed by `#ifdef USE_QPL_COMPRESSION`, as it is directly included in [CompressedReadBufferBase.cpp](https://github.com/pingcap/tiflash/pull/7364/files#diff-7d863f62031fa22aa0c5a5cad85ced3bf9b5344fb09ef96c27c023a0cf4eab21).

Reply:
Done，thanks.